drivers/mtd/mtd_partition.c

```diff
@@ -856,6 +912,8 @@ FAR struct mtd_dev_s *mtd_partition(FAR struct mtd_dev_s *mtd,
   part->child.bwrite = part_bwrite;
   part->child.read   = mtd->read ? part_read : NULL;
   part->child.ioctl  = part_ioctl;
+  part->child.isbad  = part_isbad;
+  part->child.markbad  = part_markbad;
```

Comment:
Please align the assignment of all members of `part->child`

Reply:
Done.